taskflow_test.go

```diff
@@ -508,7 +508,8 @@ func Test_wd_param(t *testing.T) {
 	flow := &goyek.Taskflow{}
 	beforeDir, err := os.Getwd()
 	requireEqual(t, err, nil, "should get work dir before the taskflow")
-	dir := tempDir(t)
+	dir, cleanup := tempDir(t)
```

Comment:
Why don't you use https://golang.org/pkg/testing/#T.TempDir.
In this case, `go test` takes care of the cleanup.

Reply:
I did use it initially but the backwards compatibility builds started failing. The method was introduced in Go 1.15. I prefer to keep the backwards compatibility as long as it is not expensive to maintain. Which is not so far.